server2/Arcadia.Assistant/Arcadia.Assistant.WorkHoursCredit/WorkHoursCredit.cs

```diff
@@ -80,6 +80,15 @@ public async Task<WorkHoursChange[]> GetCalendarEventsAsync(EmployeeId employeeI
             return events;
         }
 
+        public async Task<Dictionary<EmployeeId, WorkHoursChange[]>> GetCalendarEventsByEmployeeMapAsync(EmployeeId[] employeeIds, CancellationToken cancellationToken)
+        {
+            using var ctx = this.dbFactory();
+            var events = await this.QueryCalendarEvents(ctx.Value.ChangeRequests, x => employeeIds.Any(id => id.Value == x.EmployeeId))
+                .ToArrayAsync(cancellationToken);
+
+            return events.GroupBy(x=> x.EmployeeId).ToDictionary(x => x.Key, x=> x.ToArray());
```

Comment:
Code is not formatted.

Reply:
Done

Reply:
Not done. Code is still not formatted.
> x=> x.EmployeeId

Should be a space before an arrow.

Reply:
Don't you use R#?